### PR TITLE
Hometask 6

### PR DESCRIPTION
**Pull request by Hiblton**
in progress

### CODE REVIEW

File: src/components/MovieDetails/MovieDetails.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import PropTypes from "prop-types";
 import styles from "./MovieDetails.module.scss";
 import { Title } from "./../Title/Title";
```
Comment (i9or) [inline src/components/MovieDetails/MovieDetails.js:3]:
```suggestion
import { Title } from "../Title/Title";
```

File: src/components/MovieDetails/MovieDetails.js [hunk 2]
```diff
@@ -14,10 +14,12 @@ const MovieDetails = ({ movie }) => {
         <div className={styles.description}>
           <div className={styles.inlineFlex}>
             <Title title={movie?.title}></Title>
```
Comment (i9or) [inline src/components/MovieDetails/MovieDetails.js:16]:
```suggestion
            <Title title={movie?.title} />
```

File: src/components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -14,10 +14,12 @@ const MovieDetails = ({ movie }) => {
...
-            <div className={styles.rating}>7.3</div>
+            {movie?.rating && (
+              <div className={styles.rating}>{movie?.rating}</div>
+            )}
           </div>
           <span className={styles.movieGenre}>{movie?.genre}</span>
-          <div className={`${styles.inlineFlex} ${styles.dateAndRintime}`}>
+          <div className={`${styles.inlineFlex} ${styles.dateAndRuntime}`}>
             <div className={styles.releaseDate}>
```
Comment (i9or) [inline src/components/MovieDetails/MovieDetails.js:23]: There's no class `releaseDate` in the `MovieDetails.module.scss`.

**Review by i9or — APPROVED** [review]
LGTM

**Comment by Hiblton** [comment]
Thank you for review!

**Pull request merged**